medusa/restore_cluster.py

```diff
@@ -349,20 +349,21 @@ def _build_restore_cmd(self):
         verify_option = '--no-verify'
 
         # %s placeholders in the below command will get replaced by pssh using per host command substitution
-        command = 'mkdir -p {work}; cd {work} && medusa-wrapper sudo medusa --fqdn=%s -vvv restore-node ' \
+        command = 'mkdir -p {work}; cd {work} && medusa-wrapper {sudo} medusa --fqdn=%s -vvv restore-node ' \
                   '{in_place} {keep_auth} %s {verify} --backup-name {backup} --temp-dir {temp_dir} ' \
                   '{use_sstableloader} {keyspaces} {tables}' \
             .format(work=self.work_dir,
+                    sudo='sudo' if medusa.utils.evaluate_boolean(self.config.cassandra.use_sudo) else '',
                     in_place=in_place_option,
                     keep_auth=keep_auth_option,
                     verify=verify_option,
                     backup=self.cluster_backup.name,
                     temp_dir=self.temp_dir,
-                    use_sstableloader='--use-sstableloader' if self.use_sstableloader is True else '',
+                    use_sstableloader='--use-sstableloader' if self.use_sstableloader else '',
                     keyspaces=keyspace_options,
                     tables=table_options)
 
-        logging.debug('Preparing to restore on all nodes with the following command {}'.format(command))
+        logging.info('Preparing to restore on all nodes with the following command: {}'.format(command))
```

Comment:
I think it's useful to have an info log here but I don't have a strong opinion on it though

Reply:
I'd keep this as `debug`, it's easy to overload the logs with information (and I think we may already have some cleanup to do).